independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test/resource/basic/resource/ResponseInfoResource.java

```diff
@@ -18,7 +18,7 @@ public class ResponseInfoResource {
 
     @Path("/simple")
     @GET
-    public String get(@QueryParam("abs") String abs) {
+    public boolean get(@QueryParam("abs") String abs) {
```

Comment:
Should we add a test instead of changing an existing one?

Or this is safe?

Reply:
It's totally safe because the response itself was not being used anywhere and now it is.

I also checked that with this test the previous version of the code was failing